docs/guides/security/firewalld-beginners.md

```diff
@@ -15,14 +14,14 @@ Of course, if you're here, you probably have a better idea what a firewall is an
 
 In other words, my fellow nerds should be aware that there will be a lot of explanations incoming.
 
-So let's talk about what we're here for. `firewalld` is the default firewall app packaged with Rocky Linux, and it's designed to be pretty simple to use. You just need to know a little bit about how firewalls work, and not be afraid to use the command line.
+So, let's talk about what we're here for. `firewalld` is the default firewall app packaged with Rocky Linux, and it's designed to be pretty simple to use. You just need to know a little bit about how firewalls work, and not be afraid to use the command line.
 
 Here you'll learn:
 
 * The very basics of how `firewalld` works
 * How to use `firewalld` to restrict or allow incoming and outgoing connections
 * How to allow only people from certain IP addresses or places to log into your machine remotely
-* How to manage some `firewalld`-specific features like Zones.
+* How to manage some `firewalld`-specific features like Zones .
```

Comment:
Why did you add a space between "Zones" and the period? Did you mean to remove the period altogether? I'm fine with that.